### DISCUSSION TITLE
cannot install the latest version of packages

### DISCUSSION THREAD

**Discussion by I-AM-A-NOOB** [ScoopInstaller/Scoop#5428] [Help and Questions]
Some issue are similar to this, but I think they are not the same. 
I wanted to install package ```gcc```, so I search the package in ```scoop.sh```. 
The website shown that the latest version of the package in scoop is 11.2.0
But when I ran ```scoop search gcc```:
```
>scoop search gcc
'main' bucket:
    avr-gcc (10.2.0)
    gcc (9.3.0-2)
```
I check the json file on [GitHub](github.com/ScoopInstaller/Main/blob/master/bucket/gcc.json), and it also showned that the latest version is 11.2.0. 
But after I cloned the responsive, and replace the bucket file in the user folder, it returned ```gcc (9.3.0-2)```again.

**Comment by rashil2000** [comment]
You are using an extremely outdated version of Scoop. Please run `scoop update` first.

**Reply by I-AM-A-NOOB** [reply]
did not help.

**Reply by rashil2000** [reply]
What's the output?

**Reply by I-AM-A-NOOB** [reply]
```
>scoop update
Updating Scoop...
Updating 'main' bucket...
Scoop was updated successfully!
>scoop search gcc
'main' bucket:
    avr-gcc (10.2.0)
    gcc (9.3.0-2)
```

**Reply by I-AM-A-NOOB** [reply]
I think one possible reason is that in China scoop seems to use ```gitclone.com``` to proxy downloads

**Reply by rashil2000** [reply]
> I think one possible reason is that in China scoop seems to use `gitclone.com` to proxy downloads

It doesn't do that on its own. You must have set it long time ago. What's there in ~/.config/scoop/config.json?

**Reply by I-AM-A-NOOB** [reply]
I'm going to try to install it on another computer that doesn't have scoop installed and see what will happen.
```json
{
  "last_update": "2023-03-04T10:42:20.846758+08:00",
  "scoop_repo": "https://github.com/ScoopInstaller/Scoop",
  "scoop_branch": "master",
  "lastupdate": "2023-03-14T20:45:50.4730548+08:00"
}
```

**Reply by I-AM-A-NOOB** [reply]
I just remembered that I downloaded it from another repository in China. The code in this repository may have been modified to speed up the download of the software from scoop in China. 
Sorry, it's my fault.

**Reply by rashil2000** [reply]
No worries :)

**Reply by I-AM-A-NOOB** [reply]
**I found the reason.** Some other software or git itself had modified the ```~\.gitconfig``` file, causing git to use ```gitclone.com``` to proxy download, and the repository mirrors on ```gitclone.com``` only seemed to have last year's version. I changed ```gitclone.com``` to another proxy site (git is very slow in China) and the problem was solved.

**Reply by rashil2000** [reply]
Oh, good to know. Thanks for mentioning, it might help others who are facing similar issues.
